src/TerminalIo.cpp

```diff
@@ -1,5 +1,5 @@
 #include "TerminalIo.hpp"
-
+#if TUNDRA_UNIX
```

Comment:
what happens on windows, and what does that have to do with the rest of the dynamic output business?

Reply:
this file was only built on unix before, which required specialcasing in the buildprogram.  I preferred to simplify the buildprogram to simply glob, and deal with this single case with an ifdef